proposals/channel-jamming-mitigation.md

```diff
@@ -0,0 +1,257 @@
+of a channel by sending payments that are destined to fail through a target 
+route. This attack can exhaust the resources along the target route in two ways, 
+though the difference between the two behaviors is not rigorously defined: 
+1. Quick Jamming: sending a continuous stream of payments through a route that
+   are quickly resolved, but block all of the liquidity or HTLC slots along 
+   the route.
+2. Slow Jamming: sending a slow stream of payments through a route and only 
+   failing them at the latest possible timeout, blocking all of the liquidity
+   or HTLC slots along the route for the duration.
+
+Fees are currently only paid to routing nodes on successful completion of a 
+payment, so this attack is virtually free - the attacker pays only for the 
+costs of running a node, funding channels on-chain and the opportunity cost
+of capital committed to the attack.
+
+### Threat Model
+Attackers may aim to disrupt a specific set of channels, or the network as a
+whole. We consider the following threat model, adapted from [1]: 
+* The attacker can quickly set up a set of seemingly unrelated nodes and open
+  channels to any public node in the network.
+* The attacker has an up to date view of the network's public topology. 
+* The attacker is economically rational, meaning that they will pursue an 
+  attack with the lowest cost and highest utility (and "seeing the world burn" 
+  classifies as utility).
+* The attacker has the ability to send slow or quick jams: 
+  * Quick jamming attacks will aim to fill all available htlc slots if
+    `max_accepted_htlcs` * `htlc_minimum_msat` < `max_htlc_value_in_flight_msat`,
+    or otherwise will aim to deplete channel liqudidity.
+  * The attacker has a modified LN node implementation that allows them to hold
+    HLTCs up until the point of force closure, then release them.
+* The attacker may have access to long-lived channels when the attack begins.
+
+### Mitigation
+To comprehensively mitigate jamming attacks against the Lightning Network, a
+solution needs to address both quick and slow jamming. This proposal has two 
+parts, upfront fees to address quick jamming and reputation to address slow
+jamming [2].
+
+### Quick Jamming Mitigation
+Traffic that is part of a quick jamming attack mimics the behavior of honest 
+nodes' failed payments, which makes it difficult to identify. A solution that 
+addresses quick jamming therefore *must have a trivial impact on honest 
+traffic, but be sufficient to compensate the node(s) under attack*. A solution 
+that fails to do this will only magnify an attacker's ability to disrupt the 
+network, as they can leverage the jamming mitigation itself to further degrade 
+honest traffic.
+
+An upfront fee, paid regardless of the outcome of a payment attempt, is proposed
+to economically compensate nodes for providing traffic with access to liquidity
+and slots. Simulations in [2] show that a fee of as little as 1% of the success
```

Comment:
I think the number holds as long as the simulation parameters hold. From my understanding of the paper model, some parameters are quite arbitrary (e.g the jam delay of 7 second), and beyond I think there is some assumption of periodicity of jamming and honest payment. Where the 1% unconditional fee doesn't make sense to me is in face of a jamming adversary perpetually succeeding to occupy the HTLC slots, in a way than never the success fees are paid. In that case, there is no equal compensation at all, I would say.

Additionally, I don't know about the methodological issue of substituting our own evaluation of the routing node liquidity opportunity cost, and if unconditional fees should not be floating to suit one's own evaluation of opportunity cost.